### PR TITLE
Task priority

### PR DESCRIPTION
**Pull request by aminzoda**
Possibility to set task priority (low, medium, high)...

### CODE REVIEW

**Comment by bin-umar** [comment]
> Possibility to set task priority (low, medium, high)...

Add here screens how it looks like now
image

File: main.js [hunk 1]
```diff
@@ -38,17 +38,26 @@ document.addEventListener("DOMContentLoaded", function () {
...
-          <button class="edit">Edit</button>
+          <select class="priority">
+       <option value="low" ${priority === "low" ? "selected" : ""}>Low</option>
+       <option value="medium" ${
+         priority === "medium" ? "selected" : ""
+       }>Medium</option>
+       <option value="high" ${
+         priority === "high" ? "selected" : ""
+       }>High</option>
+     </select>
```
Comment (bin-umar) [inline main.js]: - To enhance readability of code, move this piece of code to a separate function or at least block.
- Create `priorities` array and use cycle for creating option-s

Comment (bin-umar) [reply main.js]: Add different colors for each priority:
- Red for `high`
- Yellow for `medium`
- Blue for `low`

**Review by bin-umar — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (aminzoda) [reply main.js]: I am corrected these errors.

**Comment by bin-umar** [comment]
Let's redesign a bit. 
- Use flags for priorities 
Screenshot 2024-07-31 at 14 49 18

- Change colour of checkbox, not a text label
- Sort tasks by priority (higher should be in top)

**Review by bin-umar — APPROVED** [review]

**Pull request merged**